src/ripple/protocol/ErrorCodes.h

```diff
@@ -91,9 +91,9 @@ enum error_code_i
     rpcACT_MALFORMED         = 35,
     rpcALREADY_MULTISIG      = 36,
     rpcALREADY_SINGLE_SIG    = 37,
-    rpcBAD_FEATURE           = 38,
-    // unused                  39,
-    // unused                  40,
+    // unused                  38,
+    // unused                  49,
```

Comment:
should be 39

Reply:
<Slaps Forehead> Doh!  Thanks.  Another `[FOLD]` coming soon...